File: 1-Iniciante/python/1035_teste-de-selecao.py
# Leia 4 valores inteiros A, B, C e D. A seguir, se B for maior do que C e se D for maior do que A, 
# e a soma de C com D for maior que a soma de A e B e se C e D, ambos, forem positivos e se a variável A 
# for par escrever a mensagem "Valores aceitos", senão escrever "Valores nao aceitos".

n = input().split()
A, B, C, D = int(n[0]), int(n[1]), int(n[2]), int(n[3]),  
if B > C and D > A and C + D > A + B and C > 0 and D > 0 and A % 2 == 0:
    print('Valores aceitos')
else:
    print('Valores nao aceitos')

File: 1-Iniciante/javascript/1035_teste-de-selecao.js
var entrada = require("fs").readFileSync("/dev/stdin", "utf8");
informado = entrada.split(' ')
var A = parseInt(informado[0])
var B = parseInt(informado[1])
var C = parseInt(informado[2])
var D = parseInt(informado[3])

if (B > C && D > A && C + D > A + B && C > 0 && D > 0 && A % 2 === 0) {
    console.log("Valores aceitos");
} else {
    console.log("Valores nao aceitos");
}